### PR TITLE
WFP cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VI Instrument Cal CSVs [DT-219]](https://trello.com/c/CkLoKMMl/219-py-vi-instrument-cal-csvs-dt-219)

File: calibration/FLORTK/CGINS-FLORTK-01537__20230203.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+1537,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1537_Calibration_2023-02-03.zip > BBFL2-1537.dev
+1537,CC_dark_counts_cdom,50,
+1537,CC_dark_counts_chlorophyll_a,50,
+1537,CC_dark_counts_volume_scatter,51,
+1537,CC_depolarization_ratio,0.039,
+1537,CC_measurement_wavelength,700,
+1537,CC_scale_factor_cdom,9.587E-02,
+1537,CC_scale_factor_chlorophyll_a,1.207E-02,
+1537,CC_scale_factor_volume_scatter,3.183E-06,
+1537,CC_scattering_angle,124,
```
Comment (reedan88) [inline calibration/FLORTK/CGINS-FLORTK-01537__20230203.csv:11]: Looks good.

File: calibration/FLORTK/CGINS-FLORTK-01538__20230203.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+1538,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1538_Calibration_2023-02-03.zip > BBFL2-1538.dev
+1538,CC_dark_counts_cdom,51,
+1538,CC_dark_counts_chlorophyll_a,53,
+1538,CC_dark_counts_volume_scatter,52,
+1538,CC_depolarization_ratio,0.039,
+1538,CC_measurement_wavelength,700,
+1538,CC_scale_factor_cdom,9.031E-02,
+1538,CC_scale_factor_chlorophyll_a,1.215E-02,
+1538,CC_scale_factor_volume_scatter,3.247E-06,
+1538,CC_scattering_angle,124,
```
Comment (reedan88) [inline calibration/FLORTK/CGINS-FLORTK-01538__20230203.csv:11]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20466__20230213.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20466,CC_dark_offset,1.16, source_file PARAD-K_QSP2200_SN_20466_Calibration_2023-02-13.pdf
+20466,CC_scale_wet,9.33E-18,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20466__20230213.csv:3]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20525__20230213.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20525,CC_dark_offset,1.12,source_file PARAD-K_QSP2200_SN_20525_Calibration_2023-02-13.pdf
+20525,CC_scale_wet,9.41E-18,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20525__20230213.csv:3]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20541__20230213.csv [hunk 5]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20541,CC_dark_offset,0.91,source_file PARAD-K_QSP2200_SN_20541_Calibration_2023-02-13.pdf
+20541,CC_scale_wet,9.29E-18,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20541__20230213.csv:3]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20608__20230213.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20608,CC_dark_offset,1.26,source_file PARAD-K_QSP2200_SN_20608_Calibration_2023-02-13.pdf
+20608,CC_scale_wet,9.31E-18,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20608__20230213.csv:3]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20609__20230213.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20609,CC_dark_offset,1.17,source_file PARAD-K_QSP2200_SN_20609_Calibration_2023-02-13.pdf
+20609,CC_scale_wet,9.38E-18,
```
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20609__20230213.csv:3]: Looks good.

**Review by reedan88 — APPROVED** [review]
Everything looks good.

File: calibration/FLORDL/CGINS-FLORDL-02736__20230520.csv [hunk 8]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+2736,CC_angular_resolution,1.096,Constant; source_file FLORD-L_FLBBRTD_SN_2736_Calibration_2023-05-20.zip > FLBBRTD-2736.dev
+2736,CC_dark_counts_chlorophyll_a,50,
+2736,CC_dark_counts_volume_scatter,50,
+2736,CC_depolarization_ratio,0.039,Constant
+2736,CC_measurement_wavelength,700,Constant
+2736,CC_scale_factor_chlorophyll_a,0.0073,
+2736,CC_scale_factor_volume_scatter,1.820E-06,
+2736,CC_scattering_angle,140,Constant
```
Comment (nichole-arm) [inline calibration/FLORDL/CGINS-FLORDL-02736__20230520.csv:9]: looks good

File: calibration/FLORDL/CGINS-FLORDL-03282__20230520.csv [hunk 9]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+3282,CC_angular_resolution,1.096,Constant; source_file FLORD-L_FLBBRTD_SN_3282_Calibration_2023-05-20.zip > FLBBRTD-3282.dev
+3282,CC_dark_counts_chlorophyll_a,49,
+3282,CC_dark_counts_volume_scatter,45,
+3282,CC_depolarization_ratio,0.039,Constant
+3282,CC_measurement_wavelength,700,Constant
+3282,CC_scale_factor_chlorophyll_a,0.0072,
+3282,CC_scale_factor_volume_scatter,1.712E-06,
+3282,CC_scattering_angle,140,Constant
```
Comment (nichole-arm) [inline calibration/FLORDL/CGINS-FLORDL-03282__20230520.csv:9]: looks good

File: calibration/FLORDL/CGINS-FLORDL-03283__20230520.csv [hunk 10]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+3283,CC_angular_resolution,1.096,Constant; source_file FLORD-L_FLBBRTD_SN_3283_Calibration_2023-05-20.zip > FLBBRTD-3283.dev
+3283,CC_dark_counts_chlorophyll_a,50,
+3283,CC_dark_counts_volume_scatter,42,
+3283,CC_depolarization_ratio,0.039,Constant
+3283,CC_measurement_wavelength,700,Constant
+3283,CC_scale_factor_chlorophyll_a,0.0070,
+3283,CC_scale_factor_volume_scatter,1.776E-06,
+3283,CC_scattering_angle,140,Constant
```
Comment (nichole-arm) [inline calibration/FLORDL/CGINS-FLORDL-03283__20230520.csv:9]: looks good

File: calibration/FLORTK/CGINS-FLORTK-00830__20230320.csv [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+830,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_830_Calibration_2023-03-20.zip > BBFL2-830.dev but cal date taken from Char pdf
+830,CC_dark_counts_cdom,44,
+830,CC_dark_counts_chlorophyll_a,51,
+830,CC_dark_counts_volume_scatter,47,
+830,CC_depolarization_ratio,0.039,
+830,CC_measurement_wavelength,700,
+830,CC_scale_factor_cdom,9.050E-02,
+830,CC_scale_factor_chlorophyll_a,1.237E-02,
+830,CC_scale_factor_volume_scatter,2.652E-06,
+830,CC_scattering_angle,124,
```
Comment (nichole-arm) [inline calibration/FLORTK/CGINS-FLORTK-00830__20230320.csv:11]: looks good

File: calibration/FLORTK/CGINS-FLORTK-01029__20230203.csv [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+1029,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1029_Calibration_2023-02-03.zip > BBFL2-1029.dev
+1029,CC_dark_counts_cdom,48,
+1029,CC_dark_counts_chlorophyll_a,49,
+1029,CC_dark_counts_volume_scatter,50,
+1029,CC_depolarization_ratio,0.039,
+1029,CC_measurement_wavelength,700,
+1029,CC_scale_factor_cdom,1.070E-01,
+1029,CC_scale_factor_chlorophyll_a,1.158E-02,
+1029,CC_scale_factor_volume_scatter,3.366E-06,
+1029,CC_scattering_angle,124,
```
Comment (nichole-arm) [inline calibration/FLORTK/CGINS-FLORTK-01029__20230203.csv:11]: looks good

File: calibration/FLORTK/CGINS-FLORTK-01101__20230203.csv [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+1101,CC_angular_resolution,1.076, source_file FLORT-K_BBFL2_SN_1101_Calibration_2023-02-03.zip > BBFL2-1101.dev
+1101,CC_dark_counts_cdom,50,
+1101,CC_dark_counts_chlorophyll_a,51,
+1101,CC_dark_counts_volume_scatter,51,
+1101,CC_depolarization_ratio,0.039,
+1101,CC_measurement_wavelength,700,
+1101,CC_scale_factor_cdom,1.015E-01,
+1101,CC_scale_factor_chlorophyll_a,1.248E-02,
+1101,CC_scale_factor_volume_scatter,3.253E-06,
+1101,CC_scattering_angle,124,
```
Comment (nichole-arm) [inline calibration/FLORTK/CGINS-FLORTK-01101__20230203.csv:11]: looks good

File: calibration/FLORTK/CGINS-FLORTK-01537__20230203.csv (hunk 1 above)
Comment (nichole-arm) [inline calibration/FLORTK/CGINS-FLORTK-01537__20230203.csv:11]: looks good

File: calibration/FLORTK/CGINS-FLORTK-01538__20230203.csv (hunk 2 above)
Comment (nichole-arm) [inline calibration/FLORTK/CGINS-FLORTK-01538__20230203.csv:11]: looks good

File: calibration/PARADK/CGINS-PARADK-20466__20230213.csv (hunk 3 above)
Comment (nichole-arm) [inline calibration/PARADK/CGINS-PARADK-20466__20230213.csv:3]: looks good

File: calibration/PARADK/CGINS-PARADK-20525__20230213.csv (hunk 4 above)
Comment (nichole-arm) [inline calibration/PARADK/CGINS-PARADK-20525__20230213.csv:3]: looks good

File: calibration/PARADK/CGINS-PARADK-20541__20230213.csv (hunk 5 above)
Comment (nichole-arm) [inline calibration/PARADK/CGINS-PARADK-20541__20230213.csv:3]: looks good

File: calibration/PARADK/CGINS-PARADK-20608__20230213.csv (hunk 6 above)
Comment (nichole-arm) [inline calibration/PARADK/CGINS-PARADK-20608__20230213.csv:3]: looks good

File: calibration/PARADK/CGINS-PARADK-20609__20230213.csv (hunk 7 above)
Comment (nichole-arm) [inline calibration/PARADK/CGINS-PARADK-20609__20230213.csv:3]: looks good

**Review by nichole-arm — APPROVED** [review]
all looks good!

**Pull request merged**